panels/config/zwave/zwave-log.html

```diff
@@ -50,15 +66,28 @@
         type: String,
         value: 'Refresh to pull log'
       },
+
+      numLogLines: {
+        type: Number
+      },
     };
   }
 
   refreshLog() {
     this.ozwLogs = 'Loading ozw log...';
-    this.hass.callApi('GET', 'zwave/ozwlog')
-      .then((info) => {
-        this.ozwLogs = info;
-      });
+    if (this.numLogLines === 0) {
+      this.hass.callApi('GET', 'zwave/ozwlog')
+        .then((info) => {
+          this.ozwLogs = info;
+        });
+    } else {
+      this.hass.callApi('GET', 'zwave/ozwlog')
+        .then((info) => {
+          var lines = info.split('\n');
```

Comment:
We should not limit log lines in the frontend but instead add support to the backend